zipline/_protocol.pyx

```diff
@@ -21,6 +21,28 @@ from cpython cimport bool
 
 from zipline.assets import Asset
 
+
+class assertKeywords(object):
```

Comment:
This feels like it belongs with the other pre-processors in [input_validation.py] (https://github.com/quantopian/zipline/blob/a3dbf75/zipline/utils/input_validation.py).  Should we have a cython decorator that wraps something there?